src/sentry/static/sentry/app/views/events/searchBar.jsx

```diff
@@ -1,31 +1,31 @@
 import {ClassNames} from '@emotion/core';
+import assign from 'lodash/assign';
 import flatten from 'lodash/flatten';
+import isEqual from 'lodash/isEqual';
 import memoize from 'lodash/memoize';
+import omit from 'lodash/omit';
 import PropTypes from 'prop-types';
 import React from 'react';
-import isEqual from 'lodash/isEqual';
 
 import {NEGATION_OPERATOR, SEARCH_TYPES, SEARCH_WILDCARD} from 'app/constants';
 import {defined} from 'app/utils';
 import {fetchTagValues} from 'app/actionCreators/tags';
 import SentryTypes from 'app/sentryTypes';
 import SmartSearchBar from 'app/components/smartSearchBar';
+import {FIELDS, TRACING_FIELDS} from 'app/utils/discover/fields';
 import withApi from 'app/utils/withApi';
 import withTags from 'app/utils/withTags';
 
-const tagToObjectReducer = (acc, name) => {
-  acc[name] = {
-    key: name,
-    name,
-  };
-  return acc;
-};
-
 const SEARCH_SPECIAL_CHARS_REGEXP = new RegExp(
   `^${NEGATION_OPERATOR}|\\${SEARCH_WILDCARD}`,
   'g'
 );
 
+const FIELD_TAGS = Object.keys(FIELDS).reduce((acc, item) => {
+  acc[item] = {key: item, name: item};
+  return acc;
+}, {});
```

Comment:
I've been using `Object.fromEntries` recently, I think it's maybe a bit clearer?

```suggestion
const FIELD_TAGS = Object.fromEntries(
Object.keys(FIELDS).map(item => [item, {key: item, name: item}])
);
```